nestkernel/nest.h

```diff
@@ -82,6 +82,9 @@ ArrayDatum get_connections( const DictionaryDatum& dict );
 
 void simulate( const double& t );
 void resume_simulation();
+void run( const double& t );
+void prepare();
+void cleanup();
```

Comment:
prepare -> run -> run -> ... -> run -> cleanup
resume_simulation has a different function, to run a simulation after cleanup. But it should be checked, as you point out. I would, however, make that another issue -- I wouldn't remove any functionality until we have this functionality in.